packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -434,10 +434,18 @@ mixin SchedulerBinding on BindingBase {
   }
 
   // Scheduled by _ensureEventLoopCallback.
-  void _runTasks() {
+  Future<void> _runTasks() async {
     _hasRequestedAnEventLoopCallback = false;
     if (handleEventLoopCallback())
       _ensureEventLoopCallback(); // runs next task when there's time
+    else {
```

Comment:
nit: add `{ }` to the body of the `if` as well for consistency.

Reply:
Also: This `else` and the next `if` could be combined.

Reply:
Yeah,I will fix these issues in the next submission